style.css

```diff
@@ -515,3 +941,461 @@ body {
   }
 }
 
+@media (min-width: 992px) {
+  .middle-section__clutch {
+    padding: 0 61px 0 55px;
+    -webkit-box-sizing: border-box;
+    box-sizing: border-box;
+    min-height: 556px;
+    margin-top: 14px;
+  }
+
+  .middle-section__title {
+    margin-top: 0;
+    margin-bottom: 16px;
+    padding-top: 44px;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 40px;
+    line-height: 41px;
+  }
+
+  .middle-section__subtitle {
+    margin-top: 0;
+    margin-bottom: 60px;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 16px;
+    line-height: 16px;
+    letter-spacing: .07em;
+  }
+
+  .middle-section__desription {
+    margin-top: 0;
+    margin-bottom: 43px;
+    opacity: .5;
+    line-height: 21px;
+    font-size: 15px;
+    font-weight: normal;
+  }
+
+  .middle-section__btn_other {
+    position: absolute;
+    right: 50px;
+    top: 95%;
+    width: 478px;
+    cursor: pointer;
+    text-decoration: none;
+    -webkit-box-pack: justify;
+    -ms-flex-pack: justify;
+  }
+
+  .middle-section__btn_busket {
+    width: 253px;
+    height: 44px;
+    padding-right: 10px;
+    -webkit-box-sizing: border-box;
+    box-sizing: border-box;
+    padding-top: 14px;
+  }
+
+  .middle-section__colors-and-price {
+    min-width: 360px;
+    margin-top: 0;
+    margin-bottom: 54px;
+  }
+
+  .middle-section__form-colors {
+    width: 161px;
+    height: 39px;
+  }
+
+  .form__colors-item::before {
+    left: 0;
+    width: 30px;
+    height: 30px;
+  }
+
+  input[type="radio"]:checked + .form__colors-item::before {
+    width: 38px;
+    height: 38px;
+    -webkit-transform: translate(-4px, -9%);
+    -ms-transform: translate(-4px, -9%);
+    transform: translate(-4px, -9%);
+  }
+
+  .middle-section__arrow {
+    cursor: pointer;
+    position: relative;
+    left: 240px;
+    bottom: 25px;
+  }
+
+  .middle-section__arrow div {
+    position: relative;
+    top: 18px;
+    width: 240px;
+    height: 2px;
+    background-color: #000;
+    display: block;
+  }
+
+  .middle-section__arrow div::after {
+    content: '';
+    position: absolute;
+    width: 7px;
+    height: 1px;
+    top: -3px;
+    right: 0;
+    background-color: #000;
+    transform: rotate(45deg);
+  }
+
+  .middle-section__arrow div::before {
+    content: '';
+    position: absolute;
+    width: 7px;
+    height: 1px;
+    top: 4px;
+    right: 0;
+    background-color: #000;
+    box-shadow: 0 3px 5px rgba(0, 0, 0, .2);
+    transform: rotate(-45deg);
+  }
+
+  .quantity,
+  .middle-section__btn_other {
+    display: block;
+    color: #000;
+  }
+
+  .quantity {
+    width: 100px;
+    height: 44px;
+    padding-top: 12px;
+    margin-right: 15px;
+    -webkit-box-sizing: border-box;
+    box-sizing: border-box;
+    border: 1.5px solid #000000;
```

Comment:
Codacy found an issue: [Expected "#000000" to be "#000" (color-hex-length)](https://app.codacy.com/gh/WebHeroSchool/locoalin/pullRequest?prid=5835770)